README.md

```diff
@@ -1,28 +1,47 @@
-# `@ubiquity/ts-template`
+# `@ubiquibot/command-query-user`
 
-This template repository includes support for the following:
+This package helps users querying their information, from they wallet address, label access control or current XP.
 
-- TypeScript
-- Environment Variables
-- Conventional Commits
-- Automatic deployment to Cloudflare Pages
+## Usage
 
-## Testing
-
-### Cypress
+The following commands are allowed:
 
-To test with Cypress Studio UI, run
+```shell
+/query @user
+```
 
+## Running locally
+### Worker
+Start the Worker by running
 ```shell
-yarn cy:open
+yarn dev
 ```
 
-Otherwise to simply run the tests through the console, run
+### Make requests
+To trigger the worker, `POST` requests should be made to http://localhost:4000 with a `Content-Type: application/json`
+header and a body looking like
+```json
+{
+  "stateId": "",
+  "eventName": "",
+  "eventPayload": "",
+  "settings": "",
+  "ref": ""
+}
+```
+For convenience, you can find an `.http` file with a valid request [here](/tests/http/request.http).
 
-```shell
-yarn cy:run
+## Configuration
+A valid configuration can be like:
+
+```yaml
+- plugin: https://ubiquibot-command-query-user.ubq.fi
+  type: github
```

Comment:
I think we will have a different type for workers plugin or maybe it can be implicitly inferred

Reply:
Remove type. It should be inferred by the protocol prefix (or lack thereof)

Reply:
Opened related ticket on the Kernel https://github.com/ubiquity/ubiquibot-kernel/issues/45